ggml/src/ggml-vulkan/vulkan-shaders/dequant_funcs.comp

```diff
@@ -1,6 +1,9 @@
 #if !defined(DATA_A_F32) && !defined(DATA_A_F16)
 #extension GL_EXT_shader_explicit_arithmetic_types_int8 : require
 #endif
+#if defined(DATA_A_IQ1_M)
+#extension GL_EXT_shader_explicit_arithmetic_types_float16 : require
+#endif
```

Comment:
This extension shouldn't be used due to hardware restrictions, otherwise IQ1_M is not going to work on hardware without 16-bit support.

Reply:
Removed